ios/AirPushNotification/CTAppController+APNAdditions.m

```diff
@@ -20,8 +20,11 @@
 @implementation CTAppController (APNAdditions)
 
 + (void)load {
-    
+    if([UNUserNotificationCenter class] != nil) {
```

Comment:
right, it's only iOS 10 - I could have checked the version number too but this has the same effect